website/docs/how-to/how-to-add-strategy-constraints.md

```diff
@@ -2,11 +2,9 @@
 title: How to add strategy constraints
 ---
 
-:::info Availability
-Strategy constraints are available to Unleash Pro and Enterprise users.
-:::
+:::info Availability Strategy constraints are available to Unleash Pro and Enterprise users. :::
```

Comment:
This is something that prettier does that breaks the layout, so we need to add some spaces like this:

```suggestion
:::info Availability 

Strategy constraints are available to Unleash Pro and Enterprise users. 

:::
```